modules/ROOT/pages/custom-config.adoc

```diff
@@ -0,0 +1,157 @@
+
+= Using custom configuration
+
+== Accessing the RHTAP cluster using `oc` or `kubectl`
+
+Follow the procedure for accessing the RHTAP cluster via the `oc` command line
+client link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/getting-started/getting_started_in_cli/[as
+described here].
+
+You can then view the integration test scenario CRs, for example:
+
+[,bash,subs="+quotes"]
+----
+$ oc get integrationtestscenario
+$ oc get integrationtestscenario __<testname>__ -o yaml
+----
+
+== Adding the policy configuration parameter
+
+A convenient method of modifying a CR is to use the `oc edit` command.
+
+Determine the name of the integration test you want to modify, then run the following:
+
+[,bash,subs="+quotes"]
+----
+$ oc edit integrationtestscenario __<testname>__
+----
+
+Find the `spec` key and add `params` underneath it like this:
+
+[,yaml,subs="+quotes"]
+----
+…
+spec:
+  params:
+    - name: POLICY_CONFIGURATION
+      value: github.com/enterprise-contract/config//slsa3
+…
+----
+
+The `params` key should be at the same indent level as `application` and
+`contexts` underneath the `spec` key.
+
+Before saving the YAML file, to workaround a problem with the API versions
+present in the cluster, you also need to also modify the API version to change
+`v1beta1` to `v1alpha1` in the first line of the YAML file.
+
+[,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+----
+
+Save the file to update the CR in the cluster.
+
+NOTE: The config file specified in the above example is
+link:https://github.com/enterprise-contract/config/blob/main/slsa3/policy.yaml[this
+one]. There are some other
+link:https://github.com/enterprise-contract/config[predefined configurations
+available] in that same repo. You can of course substitute your own git repo
+with your own customized policy file.
+
+NOTE: The `//` syntax in the git url is used to separate the git repo location
+and the subdirectory inside the git repo. EC will look for a `policy.yaml` or
+`.ec/policy.yaml` file in the directory specified, or at the top level of the git
+repo if no directory is specified.
+
+== Using an EnterpriseContractPolicy CR
+
+In the above example we set the POLICY_CONFIGURATION parameter in the
+IntegrationTestScenario CR a git url. It's also possible to set it to the name
+of a EnterpriseContractPolicy CR present in the cluster.
+
+The following procedure shows how to create such a CR and reference it in the
+POLICY_CONFIGURATION param.
+
+Create a yaml file called `policy.yaml` with the following content:
+
+[,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: EnterpriseContractPolicy
+metadata:
+  name: ec-policy
+spec:
+  description: An example custom EC policy
+  publicKey: k8s://tekton-chains/public-key
+  sources:
+    - name: Release policies
+      policy:
+        - github.com/enterprise-contract/ec-policies//policy/lib
```

Comment:
Bit awkward we need to include this...

Reply:
Yeah true. We could choose to use the bundle instead, since the bundle has it included. It also should works with just `//policy`, though we'd be fetching and parse some unrelated rego.